autode/smiles/builder.py

```diff
@@ -15,21 +15,29 @@
 
 
 class Builder(AtomCollection):
-    """3D geometry builder
+    """3D geometry builder::
```

Comment:
is the :: meaningful?

Reply:
yeah – so Sphinx renders the following as raw text

Reply:
ah fair